apps/site/assets/ts/stop/components/DepartureTimes.tsx

```diff
@@ -294,18 +296,40 @@ const DepartureTimes = ({
   const schedules = schedulesByHeadsign(schedulesForDirection);
 
   return (
-    <>
-      {Object.entries(schedules).map(([headsign, schs]) => {
-        const formattedTimes = toDisplayTime(
-          schs,
-          predictionsByHeadsign[headsign]
-            ? predictionsByHeadsign[headsign]
-            : [],
-          overrideDate
-        );
-        return departureTimeRow(headsign, formattedTimes);
-      })}
-    </>
+    <div>
+      {onClick ? (
+        <div
+          className="testSample"
+          onClick={() => onClick(route, directionId, schedulesForDirection)}
+          onKeyDown={() => onClick(route, directionId, schedulesForDirection)}
+          role="presentation"
+        >
+          {Object.entries(schedules).map(([headsign, schs]) => {
+            const formattedTimes = toDisplayTime(
+              schs,
+              predictionsByHeadsign[headsign]
+                ? predictionsByHeadsign[headsign]
+                : [],
+              overrideDate
+            );
+            return departureTimeRow(headsign, formattedTimes);
+          })}
```

Comment:
question: is the `departureTimeRow` the element that should have the `onClick` functionality? The title suggests that each row should be clickable, but it seems here like the rows are wrapped in a single clickable div.

Reply:
Yeah the `onClick` should be on the [div](https://github.com/mbta/dotcom/blob/master/apps/site/assets/ts/stop/components/DepartureTimes.tsx#L221) returned by the departure time row.  There is currently a placeholder [button](https://github.com/mbta/dotcom/blob/master/apps/site/assets/ts/stop/components/DepartureTimes.tsx#L255) on the row around the chevron.  That button should be replaced to make the whole row clickable, as stated in the TODO above the button